src/rust/engine/process_execution/Cargo.toml

```diff
@@ -10,25 +10,25 @@ async-trait = "0.1"
 walkdir = "2"
 async_semaphore = { path = "../async_semaphore" }
 bazel_protos = { path = "bazel_protos" }
-bytes = "0.5"
+bytes = "1.0"
 derivative = "2.1.1"
 grpc_util = { path = "../grpc_util" }
 fs = { path = "../fs" }
 futures = "0.3"
 hashing = { path = "../hashing" }
 libc = "0.2.39"
 log = "0.4"
-nails = "0.11"
+nails = { git = "https://github.com/tdyas/nails", rev = "889abe4e63a27899d2f6ce50adc8489b289a19b0" }
```

Comment:
Update whenever `nails` is released with Tokio 1.0 support.

Reply:
Now released as `0.12.0`.